microprofile/opentracing-extension/src/main/java/org/wildfly/extension/microprofile/opentracing/SubsystemExtension.java

```diff
@@ -42,10 +43,17 @@
     private static final String RESOURCE_NAME = SubsystemExtension.class.getPackage().getName() + ".LocalDescriptions";
 
     protected static final ModelVersion VERSION_1_0_0 = ModelVersion.create(1, 0, 0);
-    private static final ModelVersion CURRENT_MODEL_VERSION = VERSION_1_0_0;
+    protected static final ModelVersion VERSION_2_0_0 = ModelVersion.create(2, 0, 0);
+    private static final ModelVersion CURRENT_MODEL_VERSION = VERSION_2_0_0;
+
+    private static final PersistentResourceXMLParser PARSER = SubsytemParser_2_0.INSTANCE;
+    public static final String NAMESPACE = SubsytemParser_2_0.OPENTRACING_NAMESPACE;
 
     static ResourceDescriptionResolver getResourceDescriptionResolver(final String... keyPrefix) {
-        return getResourceDescriptionResolver(true, keyPrefix);
+        String[] prefixes = new String[keyPrefix.length + 1];
+        prefixes[0] = SUBSYSTEM_NAME;
+        System.arraycopy(keyPrefix, 0, prefixes, 1, keyPrefix.length);
```

Comment:
This seems a bit unnecessary.  Any way to avoid it?

Reply:
Looking into it but this is not relevant to this PR :)

Reply:
no part of this PR ;)  but fixing that